Assets/MixedRealityToolkit-SDK/StandardAssets/Controllers/RiftControllers/Left_Touch_Controller_Prefab.prefab

```diff
@@ -0,0 +1,1102 @@
+%YAML 1.1
```

Comment:
Should these prefabs have materials?

![image](https://user-images.githubusercontent.com/3580640/43290592-63a61276-90e3-11e8-9a11-8750e89cd992.png)

Reply:
Hmm, I'll check